### PR TITLE
Added MatuusOS

### PR DESCRIPTION
**Pull request by ghost**
## Description

Added MatuusOS

### CODE REVIEW

**Comment by C47F15H** [comment]
As dylanaraps said, colour blocks and other characters must not be included for the ASCII logos.

**Comment by MatusModder** [comment]
ok, i will keep that in mind.

**Comment by hykilpikonna** [comment]
I'm dying from OCD 😭 
Please center it...

File: neofetch [hunk 1]
```diff
@@ -6307,7 +6307,38 @@ ${c2}        .oossssso-`${c3}//${c1}`/ossssss+`
...
+░░░░░░░░░░░░░░██████████▓▒██████████░░░░░░░░░░░░
+░░░░░░░░░░░░░░▓████████████████████▒░░░░░░░░░░░░
+░░░░░░░░░░░░░░░███████████████████▓░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░█████████████████▓░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░▓██████████████▒░░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░░░▒▓████████▓░░░░░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
```
Comment (hykilpikonna) [inline neofetch:6337]:
```suggestion
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░▒▓▓████▓▒▒░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░▒▓████████████▓░░░░░░░░░░░░░░
░░░░░░░░░░░░▓████████████████▒░░░░░░░░░░░░
░░░░░░░░░░░▓██████████████████▒░░░░░░░░░░░
░░░░░░░░░░▓████▒▓███████▓▓█████░░░░░░░░░░░
░░░░░░░░░░██████▓░▓████▒░██████▓░░░░░░░░░░
░░░░░░░░░▒███████▓░▒▓▒░░████████░░░░░░░░░░
░░░░░░░░░▒█████████▒░░░█████████░░░░░░░░░░
░░░░░░░░░░██████████▓▒██████████░░░░░░░░░░
░░░░░░░░░░▓████████████████████▒░░░░░░░░░░
░░░░░░░░░░░███████████████████▓░░░░░░░░░░░
░░░░░░░░░░░░█████████████████▓░░░░░░░░░░░░
░░░░░░░░░░░░░▓██████████████▒░░░░░░░░░░░░░
░░░░░░░░░░░░░░░▒▓████████▓░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
```

**Comment by hykilpikonna** [comment]
I just fixed the centering and merged it into HyFetch. Looks nice!

image

**Comment by hykilpikonna** [comment]
Thank you for your contribution! 

This PR is [merged into hyfetch](https://github.com/hykilpikonna/hyfetch/commit/078ff79c) since this repo (dylanaraps/neofetch) seems no longer maintained.

[HyFetch](https://github.com/hykilpikonna/hyfetch) is a fork of neofetch with LGBTQ pride flags, but the repo also maintains an updated version of the original neofetch, addressing many pull requests that are not merged in the original repo.

Read the ["Running Updated Original Neofetch" section](https://github.com/hykilpikonna/hyfetch#running-updated-original-neofetch) for more info!

**Comment by ghost** [comment]
ok.

**Pull request closed without merging**